docs/en_US/Compression/v2_pruning_config_list.rst

```diff
@@ -0,0 +1,66 @@
+Pruning Config List Interpretation
+==================================
+
+The Keys in Config List
+-----------------------
+
+Each sub-config in the config list is a dict, and the scope of each setting (key) is only internal to each sub-config.
+If multiple sub-configs are configured with the same module, the later ones will overwrite the previous ones.
+
+op_types
+^^^^^^^^
+
+The type of the module specified in this sub-config.
+If ``op_names`` is not set in this sub-config, all modules in the model that satisfy the type will be selected.
+If ``op_names`` is set in this sub-config, the selected module should satisfy both type and name.
+
+op_names
+^^^^^^^^
+
+The name of the module specified in this sub-config.
+If ``op_types`` is set in this sub-config, the selected module should satisfy both type and name.
+
+op_partial_names
+^^^^^^^^^^^^^^^^
+
+This key is for the modules to be pruned with similar names. NNI will find all names in the model,
+find names that contain one of ``op_partial_names``, and append them into the ``op_names``.
+
+sparsity
+^^^^^^^^
+
+The sparsity ratio of each selected module.
+
+e.g., the ``sparsity`` is 0.8 means each selected module will mask 80% values on the weight.
+If ``layer_1`` (500 parameters) and ``layer_2`` (1000 parameters) are selected in this sub-config,
+then ``layer_1`` will be masked 400 parameters and ``layer_2`` will be masked 800 parameters.
+
+sparsity_per_layer
+^^^^^^^^^^^^^^^^^^
+
+Another name for ``sparsity``.
+
+total_sparsity
+^^^^^^^^^^^^^^
+
+The sparsity ratio of all selected modules, means that sparsity ratio may no longer be even between modules.
+
+e.g., the ``total_sparsity`` is 0.8 means 80% of parameters in this sub-config will be masked.
+If ``layer_1`` (500 parameters) and ``layer_2`` (1000 parameters) are selected in this sub-config,
+then ``layer_1`` and ``layer_2`` will be masked a total of 1200 parameters,
+how these total parameters are distributed between the two layers is determined by the pruning algorithm.
+
+max_sparsity_per_layer
+^^^^^^^^^^^^^^^^^^^^^^
+
+This key is usually used with ``total_sparsity``. It limits the maximum sparsity ratio of each layer.
```

Comment:
"layer" and "module" have the same meaning?

Reply:
yes, they are the same, I will unify them to `layer`

Reply:
fixed